src/pybamm_cookiecutter/entry_point.py

```diff
@@ -114,4 +122,22 @@ def __getattribute__(self, name):
               raise error
 
 #: Singleton Instance of :class:ParameterSets """
-parameter_sets = ParameterSets()
+parameter_sets = EntryPoint(group="parameter_sets")
+
+#: Singleton Instance of :class:ModelEntryPoints"""
+models = EntryPoint(group="model_entry_points")
+
+def Model(model:str):
+    """
+    Returns the loaded model object
+
+    Parameters
+    ----------
+    model : str
+        The model name or author name of the model mentioned at the model entry point.
+    Returns
+    -------
+    pybamm.model
+        Model object of the initialised model.
```

Comment:
We should add an "Examples" section here as well so that people know what the format for the model string is (e.g., "Author/Year" or whichever one we decide to use).

Reply:
I think we can do this when we have the docs ready